app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    standard_copy_command = "cp"
+    copy_command_words_total = 3
+    copy_command_error_message = "Bad Command"
+
+    normalized_command = command.split()
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()